README.md

```diff
@@ -76,189 +76,164 @@ await server.stop()
 
 ## API
 
-### `IPFSFactory` - `const f = IPFSFactory.create([options])`
-
-`IPFSFactory.create([options])` returns an object that will expose the `df.spawn` method
-
-- `options` - optional object with:
-  - `remote` bool - use remote endpoint to spawn the nodes.
-  - `port` number - remote endpoint port. Defaults to 43134.
-  - `exec` - IPFS executable path. `ipfsd-ctl` will attempt to locate it by default. If you desire to spawn js-ipfs instances in the same process, pass the ref to the module instead (e.g `exec: require('ipfs')`)
-  - `type` - the daemon type, see below the options
-    - `go` - spawn go-ipfs daemon
-    - `js` - spawn js-ipfs daemon
-    - `proc` - spawn in-process js-ipfs instance. Needs to be called also with exec. Example: `DaemonFactory.create({type: 'proc', exec: require('ipfs') })`.
-  - `IpfsClient` - A custom IPFS API constructor to use instead of the packaged one
-
-**example:** See [Usage](#usage)
-
-#### Spawn a daemon with `f.spawn([options]) : Promise`
-
-Spawn the daemon
-
-- `options` is an optional object the following properties:
-  - `init` bool (default true) or Object - should the node be initialized
-  - `initOptions` object - should be of the form `{bits: <size>}`, which sets the desired key size
-  - `start` bool (default true) - should the node be started
-  - `repoPath` string - the repository path to use for this node, ignored if node is disposable
-  - `disposable` bool (default true) - a new repo is created and initialized for each invocation, as well as cleaned up automatically once the process exits
-  - `defaultAddrs` bool (default false) - use the daemon default `Swarm` addrs
-  - `args` - array of cmd line arguments to be passed to ipfs daemon
-  - `config` - ipfs configuration options
-
-Returns a promise that resolves to:
-
-- `ipfsd` - is the daemon controller instance:
-  - `api` - a property of `ipfsd`, an instance of  [ipfs-http-client](https://github.com/ipfs/js-ipfs-http-client) attached to the newly created ipfs node
-
-**example:** See [Usage](#usage)
-
-#### Get daemon version with `f.version() : Promise`
-
-Get the version without spawning a daemon
-
-- `callback` - is a function with the signature `function(err, version)`, where version might be one of the following:
-    - if `type` is 'go' a version string like `ipfs version <version number>`
-    - if `type` is 'js' a version string like `js-ipfs version: <version number>`
-     - if `type` is 'proc' an object with the following properties:
-        - version - the ipfs version
-        - repo - the repo version
-        - commit - the commit hash for this version
-
-### Remote endpoint - `const server = IPFSFactory.createServer([options])`
-
-`IPFSFactory.createServer` starts a IPFSFactory endpoint.
+### `create([options])`
```

Comment:
This might be better named `createFactory([options])`?

Reply:
damn it was like that i just changed it back yesterday to reduce the diff for the end user.
do you think it's worth it ?

Reply:
I think it's good to be explicit.  The diff is going to be hefty anyway, right?

Reply:
In the top level api nothing actually changed just the options are different